types/slate-react/index.d.ts

```diff
@@ -116,27 +106,34 @@ export interface BasicEditorProps {
     tabIndex?: number;
 }
 
-// tsling:disable interface-over-type-literal
 export type EditorProps = BasicEditorProps & Plugin;
 
 export interface EditorState {
     schema: Schema;
     value: Value;
-    stack: Stack; // [TODO] define stack
+    stack: Stack;
 }
 
 export class Editor extends React.Component<EditorProps, EditorState> {
+    controller: Controller;
     schema: Schema;
-    value: Value;
     stack: Stack;
 
     readonly plugins: Plugin[];
 
     // Instance Methods
+    applyOperation(...args: any[]): Controller;
     blur(): void;
-    change(fn: (change: Change) => any): void;
-    change(...args: any[]): void;
+    command(...args: any[]): Controller;
     focus(): void;
+    normalize(...args: any[]): Controller;
+    operations(): Operations;
```

Comment:
this should be `readonly operations: List<Operation>` since it's a getter (and not `Operations`, which is just the apply/invert functions)